### PR TITLE
First steps

### PR DESCRIPTION
**Pull request by alexmedinasf**
Created a new app. Gave Blog name.
Initialized your project with Git.
Made sure that my project have Postgres database set up.
Built my project schema

### CODE REVIEW

File: db/migrate/20230919171611_create_comments.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.references :author, foreign_key: { to_table: :users }
```
Comment (omar-muhamad) [inline db/migrate/20230919171611_create_comments.rb]: - [ ] The foreign key between the comments table and the users table should be `user_id`, not `author_id` as you did. Please fix that. Check the given ERD below.

image

[Hint] It's supposed to be like that:
```ruby
t.references :user, foreign_key: true
```

File: db/migrate/20230919171846_create_likes.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class CreateLikes < ActiveRecord::Migration[7.0]
+  def change
+    create_table :likes do |t|
+      t.references :author, foreign_key: { to_table: :users }
```
Comment (omar-muhamad) [inline db/migrate/20230919171846_create_likes.rb]: - [ ] Same here, The foreign key between the likes table and the users table should be `user_id`, not `author_id` as you did. Please fix that. Check the given ERD below.

**Review by omar-muhamad — CHANGES_REQUESTED** [review]
# Changes Required ♻️
### Hi @alexmedinasf,

### Nice job, you are almost there.

cat-driving-serious

## (Highlights) Good Points: :+1:
- Your work is well documented. ✔
- The correct workflow is used.  ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- [ ] Please check the inline comments below ⬇️.

## [Optional] suggestions:

- Nothing to mention.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: db/migrate/20230919171846_create_likes.rb (hunk 2 above)
Comment (alexmedinasf) [reply db/migrate/20230919171846_create_likes.rb]: Thank you so much, @Omar-Muhamad , for the review. I'll work on the changes right away!

**Review by levy002 — APPROVED** [review]
## Hello @alexmedinasf 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- No linter error :heavy_check_mark: 
- You implemented the suggested changes :heavy_check_mark: 

## Status: APPROVED :fire:  :fire: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**